File: Modul 02/03/PRAK203-2410817310020-M.RizkiDinar.c
#include <stdio.h>

int main() {
    float a[6];
    scanf("%f %f %f %f %f %f",&a[0],&a[1],&a[2],&a[3],&a[4],&a[5]);
    
    float cal = (a[0] - a[1]) * ((float)a[2] / a[3]) - (a[4] + a[5]);
    
    printf("%.3f\n", cal);
    return 0;
}

File: Modul 02/03/PRAK203-2410817310020-M.RizkiDinar.py
x = []

while len(x) < 6:
    x.extend(list(map(int, input().split())))

cal = (x[0]-x[1])*(x[2]/x[3])-(x[4]+x[5])
result = round(cal, 3)

print("{:.3f}".format(result))